src/sentry/incidents/models.py

```diff
@@ -450,6 +456,7 @@ class AlertRule(Model):
     comparison_delta = models.IntegerField(null=True)
     date_modified = models.DateTimeField(default=timezone.now)
     date_added = models.DateTimeField(default=timezone.now)
+    monitor_type = models.IntegerField(default=MonitorType.CONTINUOUS)
```

Comment:
doesn't this need a separate migration?

Reply:
Ah yes - yes it would

I'm going to be adding all the model updates into this PR first though so we can get a wholistic view of what's going to change
then once we have signoff, i'll actually generate the migrations in separate PRs